dependencies.yaml

```diff
@@ -34,7 +35,22 @@ dependencies:
           - c-compiler
           - cxx-compiler
           - make
+          - sysroot_linux-64==2.17
     specific:
+      - output_types: conda
+        matrices:
+          - matrix:
+              cuda: "11.[245]"
```

Comment:
@jameslamb What do you think of this syntax? This is valid glob syntax and should work since it's handled by `fnmatch.fnmatch`. Just wanted to point it out explicitly to you since it's different than what we've been writing elsewhere.

Reply:
I thought this could lead to surprises if there were ever double-digit releases (`11.10+`), but I guess that isn't true and `[]` matches a single character.

On Python 3.10.12:

```python
import fnmatch

fnmatch.fnmatch("11.12", "11.[1245]")
# False

fnmatch.fnmatch("11.12", "11.[1245][12]")
# True

fnmatch.fnmatch("11.12", "11.[245][12]")
# False
```

Given that, I support this and think it's a nice simplification. Thanks for the `@`!